src/routes/WorkItems/components/Solutions/FinalSurvey/index.jsx

```diff
@@ -24,21 +25,10 @@ const FinalSurvey = ({ saveSurvey, onCancel, customerFeedback }) => {
 
   return (
     <div styleName="final-servey">
-      <div styleName="header">
-        <h4 styleName="title">HOW DID WE DO?</h4>
-        <p styleName="subtitle">
-          To mark this work as done, please provide feedback on your experience.
-        </p>
-        <Button
-          type={BUTTON_TYPE.ROUNDED}
-          styleName="close-btn"
-          onClick={() => onCancel(questions)}
-        >
-          <IconClose />
-        </Button>
-      </div>
 
-      <hr styleName="divider" />
+      <p styleName="subtitle">
```

Comment:
This style causes the text to have leading caps for each word:

**To Mark This Work As Done, Please Provide Feedback On Your Experience.**

While this is what the comp shows, it looks odd, violates normal writing rules, and is not what we want. Please change the style to remove the capitalization. Also, remove the period at the end of the phrase.